src/sql/finalize.sql

```diff
@@ -452,4 +452,145 @@ CREATE VIEW pg_vector_index_stat AS
          pg_am A ON A.oid = I.relam
     WHERE A.amname = 'vectors';
 
+CREATE TYPE veci8 (
```

Comment:
It'll be better if you put SQL under `-- List of xxx` so that we have same order of objects among document (https://docs.pgvecto.rs/reference/schema.html) and SQL.